### ISSUE TITLE
Error producing message for topic with protobuf schema including a map

### ISSUE THREAD

**Issue by m-kay** [provectus/kafka-ui#3674]
I configured a topic with the following protobuf schema in Kafka-UI version v0.6.2:

```proto
syntax = "proto3";

package com.example.proto;

option (version) = "2.0.0";

message SomeMessage {
  string name = 1;
  sint64 id = 2;
  map<string, string> fields = 3;
}
```
When I try to produce a message with the following content:
```json
{
   "name": "my item",
   "id": 1234,
   "fields": [
       {
          "key": "my-key",
          "value": "some-value"
       }
   ]
}
```
I get the error 
```
500 Internal Server Error
Expect a map object but found: [{"key":"my-key","value":"some-value"}]
```
and in the application log I can see the following:
```
com.google.protobuf.InvalidProtocolBufferException: Expect a map object but found: [{"key":"my-key","value":"some-value"}]
2023-04-17T07:04:50.865106218Z 	at com.google.protobuf.util.JsonFormat$ParserImpl.mergeMapField(JsonFormat.java:1676)
2023-04-17T07:04:50.865109801Z 	Suppressed: reactor.core.publisher.FluxOnAssembly$OnAssemblyException: 
2023-04-17T07:04:50.865113134Z Error has been observed at the following site(s):
2023-04-17T07:04:50.865116176Z 	*__checkpoint ⇢ Handler com.provectus.kafka.ui.controller.MessagesController#sendTopicMessages(String, String, Mono, ServerWebExchange) [DispatcherHandler]
2023-04-17T07:04:50.865120093Z 	*__checkpoint ⇢ com.provectus.kafka.ui.config.ReadOnlyModeFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865123093Z 	*__checkpoint ⇢ com.provectus.kafka.ui.config.CustomWebFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865126093Z 	*__checkpoint ⇢ org.springframework.security.web.server.authorization.AuthorizationWebFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865129468Z 	*__checkpoint ⇢ org.springframework.security.web.server.authorization.ExceptionTranslationWebFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865132551Z 	*__checkpoint ⇢ org.springframework.security.web.server.authentication.logout.LogoutWebFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865135968Z 	*__checkpoint ⇢ org.springframework.security.web.server.savedrequest.ServerRequestCacheWebFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865139093Z 	*__checkpoint ⇢ org.springframework.security.web.server.context.SecurityContextServerWebExchangeWebFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865142301Z 	*__checkpoint ⇢ org.springframework.security.web.server.context.ReactorContextWebFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865145259Z 	*__checkpoint ⇢ org.springframework.security.web.server.header.HttpHeaderWriterWebFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865148259Z 	*__checkpoint ⇢ org.springframework.security.config.web.server.ServerHttpSecurity$ServerWebExchangeReactorContextWebFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865151843Z 	*__checkpoint ⇢ org.springframework.security.web.server.WebFilterChainProxy [DefaultWebFilterChain]
2023-04-17T07:04:50.865154801Z 	*__checkpoint ⇢ org.springframework.boot.actuate.metrics.web.reactive.server.MetricsWebFilter [DefaultWebFilterChain]
2023-04-17T07:04:50.865158384Z 	*__checkpoint ⇢ HTTP POST "/api/clusters/local/topics/some-message/messages" [ExceptionHandlingWebHandler]
2023-04-17T07:04:50.865161259Z Original Stack Trace:
2023-04-17T07:04:50.865164009Z 		at com.google.protobuf.util.JsonFormat$ParserImpl.mergeMapField(JsonFormat.java:1676)
2023-04-17T07:04:50.865166926Z 		at com.google.protobuf.util.JsonFormat$ParserImpl.mergeField(JsonFormat.java:1659)
2023-04-17T07:04:50.865169801Z 		at com.google.protobuf.util.JsonFormat$ParserImpl.mergeMessage(JsonFormat.java:1502)
2023-04-17T07:04:50.865172676Z 		at com.google.protobuf.util.JsonFormat$ParserImpl.merge(JsonFormat.java:1460)
2023-04-17T07:04:50.865180176Z 		at com.google.protobuf.util.JsonFormat$ParserImpl.merge(JsonFormat.java:1340)
2023-04-17T07:04:50.865183051Z 		at com.google.protobuf.util.JsonFormat$Parser.merge(JsonFormat.java:477)
2023-04-17T07:04:50.865186009Z 		at com.provectus.kafka.ui.serdes.builtin.ProtobufFileSerde$1.serialize(ProtobufFileSerde.java:154)
2023-04-17T07:04:50.865189259Z 		at com.provectus.kafka.ui.serdes.SerdeInstance.lambda$serializer$3(SerdeInstance.java:63)
2023-04-17T07:04:50.865192176Z 		at com.provectus.kafka.ui.serdes.SerdeInstance.wrapWithClassloader(SerdeInstance.java:34)
2023-04-17T07:04:50.865194843Z 		at com.provectus.kafka.ui.serdes.SerdeInstance.lambda$serializer$4(SerdeInstance.java:63)
2023-04-17T07:04:50.865197718Z 		at com.provectus.kafka.ui.serdes.ProducerRecordCreator.create(ProducerRecordCreator.java:27)
2023-04-17T07:04:50.865200801Z 		at com.provectus.kafka.ui.service.MessagesService.sendMessageImpl(MessagesService.java:118)
2023-04-17T07:04:50.865203676Z 		at com.provectus.kafka.ui.service.MessagesService.lambda$sendMessage$8(MessagesService.java:93)
2023-04-17T07:04:50.865206593Z 		at reactor.core.publisher.MonoFlatMap$FlatMapMain.onNext(MonoFlatMap.java:125)
2023-04-17T07:04:50.865209426Z 		at reactor.core.publisher.MonoPublishOn$PublishOnSubscriber.run(MonoPublishOn.java:181)
2023-04-17T07:04:50.865212343Z 		at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:68)
2023-04-17T07:04:50.865215134Z 		at reactor.core.scheduler.SchedulerTask.call(SchedulerTask.java:28)
2023-04-17T07:04:50.865217884Z 		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2023-04-17T07:04:50.865220968Z 		at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
2023-04-17T07:04:50.865224259Z 		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2023-04-17T07:04:50.865227134Z 		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2023-04-17T07:04:50.865230009Z 		at java.base/java.lang.Thread.run(Thread.java:833)
```

**Comment by m-kay** [comment]
Is there a chance to bring this fix also into the next 0.6.x release? Or are you planning to release 0.7.0 soon?

**Comment by Haarolean** [comment]
@m-kay 0.7 is planned for 28th of April
